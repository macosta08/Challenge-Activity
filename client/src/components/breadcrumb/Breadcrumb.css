.breadcrumb {
  width: 100%;
  color: #999999;
  font-size: 10px;
  padding: 16px 0 16px 0;
}

.icon-rigth {
  font-size: 5px;
  padding: 0px 6px 0px 6px;
}

@media (min-width: 768px) {
  .breadcrumb {
    font-size: 11px;
  }
  .icon-rigth {
    font-size: 10px;
    padding: 0px 5px 0px 5px;
  }
}

@media (min-width: 1280px) {
  .breadcrumb {
    font-size: 14px;
  }
  .icon-rigth {
    font-size: 10px;
    padding: 0px 8px 0px 8px;
  }
}
