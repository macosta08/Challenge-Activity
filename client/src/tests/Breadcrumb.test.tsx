import { render, screen } from '@testing-library/react';
import { Breadcrumb } from '../components/breadcrumb/Breadcrumb';

describe('Prueba en <Breadcrumb />', () => {
  const categories = ['Hogar', 'Textiles', 'Ropa'];
  test('Prueba Snapshot', () => {
    const { container } = render(<Breadcrumb categories={categories} />);
    expect(container).toMatchSnapshot();
  });

  test('Debe mostrar el categories enviado por props', () => {
    render(<Breadcrumb categories={categories} />);

    expect('Hogar').toBeTruthy();
  });
});
