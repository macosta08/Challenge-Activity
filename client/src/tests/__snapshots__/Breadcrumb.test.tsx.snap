// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <Breadcrumb /> Prueba Snapshot 1`] = `
<div>
  <div
    class="breadcrumb"
  >
    <span>
      Hogar
      <i
        class="fas fa-chevron-right icon-rigth"
      />
    </span>
    <span>
      Textiles
      <i
        class="fas fa-chevron-right icon-rigth"
      />
    </span>
    <span>
      <b>
        Ropa
      </b>
    </span>
  </div>
</div>
`;
